spec/initializers/feature_toggles_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe 'config/initializers/feature_toggles.rb' do
+  let(:user) { build(:user) }
+
+  around do |example|
+    original = ENV['BETA_TESTERS']
+    example.run
+    ENV['BETA_TESTERS'] = original
+  end
+
+  context 'when beta_testers is ["all"]' do
+    before { ENV['BETA_TESTERS'] = 'all' }
+
+    it 'returns true' do
+      require './config/initializers/feature_toggles' # execute the initializer's code block
+
+      enabled = OpenFoodNetwork::FeatureToggle.enabled?(:customer_balance, user)
+      expect(enabled).to eq(true)
+    end
+  end
+
+  context 'when beta_testers is a list of emails' do
+    let(:other_user) { build(:user) }
+
+    before { ENV['BETA_TESTERS'] = "#{user.email}, #{other_user.email}" }
+
+    context 'and the user is in the list' do
+      it 'enables the feature' do
+        require './config/initializers/feature_toggles' # execute the initializer's code block
+
+        enabled = OpenFoodNetwork::FeatureToggle.enabled?(:customer_balance, user)
+        expect(enabled).to eq(true)
+      end
+    end
+
+    context 'and the user is not in the list' do
+      it 'disables the feature' do
+        require './config/initializers/feature_toggles' # execute the initializer's code block
+
+        enabled = OpenFoodNetwork::FeatureToggle.enabled?(:customer_balance, user)
+        expect(enabled).to eq(true)
```

Comment:
As far as I understand, you wanted to test if `the user is not in the list` (which is a case that worth testing I guess). But my understanding is that `ENV['BETA_TESTERS']` does contain the user, because of the `before` statement line#26.

Reply:
oh :hankey: very good catch @jibees . I did copy paste here and didn't update the setup for this `not in the list` context :see_no_evil:

Reply:
I rebased my commits and forced-pushed. Now the first commit should be right.